psl/builtin-connectors/src/native_type_definition.rs

```diff
@@ -0,0 +1,252 @@
+//! A [tt-muncher macro](https://veykril.github.io/tlborm/decl-macros/patterns/tt-muncher.html) for
+//! native type definitions.
+//! It also uses the [internal rule pattern](https://veykril.github.io/tlborm/decl-macros/patterns/internal-rules.html) quite extensively.
+
+#[macro_export]
+macro_rules! definition {
```

Comment:
Naming. This is a crate-wide macro, so `definition` doesn't really tell that much...

Reply:
How much more broilerplate it is to not use a macro for the definitions. It's a bit hard for me to really understand what the macro generates.

Reply:
let's rename to `native_type_definition`?

Reply:
we renamed